src/Layouts/FilterPaneRow.vala

```diff
@@ -93,15 +93,20 @@ public class Layouts.FilterPaneRow : Gtk.FlowBoxChild {
 
         Services.EventBus.get_default ().pane_selected.connect ((pane_type, id) => {
             if (pane_type == PaneType.FILTER && filter_type.to_string () == id) {
+                var css_classes = get_css_classes ();
```

Comment:
You could use `has_css_class` instead of traversing the list of classes.
https://valadoc.org/gtk4/Gtk.Widget.has_css_class.html

Reply:
Ah, that makes a lot of sense, thanks.